pkgs/by-name/br/brill/package.nix

```
@@ -0,0 +1,38 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "brill";
+  version = "4.000.073";
+
+  src = fetchzip {
+    url = "https://brill.com/fileasset/The_Brill_Typeface_Package_v_4_0.zip";
+    hash = "sha256-ugmEIkeBzD/4C9wkVfbctEtnzI8Kw+YD6KGcbk4BAf4=";
+    stripRoot = false;
+  };
+
+  installPhase = with finalAttrs; ''
+    runHook preInstall
+
+    install -Dm644 *.ttf          -t $out/share/fonts/truetype
+    install -Dm644 *agreement.pdf -t $out/share/licenses/${pname}-${version}
+    install -Dm644 *use.pdf       -t $out/share/doc/${pname}-${version}
```

```suggestion
    install -Dm644 *agreement.pdf -t $out/share/licenses/${pname}
    install -Dm644 *use.pdf       -t $out/share/doc/${pname}
```
`version` is already part of the store path.